src/services/chart/core.ts

```diff
@@ -283,30 +283,35 @@ export default abstract class Chart<T extends Record<string, any>> {
 			logger.info(`${this.name + (group ? `:${group}` : '')} (${span}): Initial commit created`);
 		}
 
+		const date = Chart.dateToTimestamp(current);
+		const lockKey = `${this.name}:${date}:${group}:${span}`;
+
+		const unlock = await getChartInsertLock(lockKey);
 		try {
+			// ロック内でもう1回チェックする
+			const currentLog = await this.repository.findOne({
+				span: span,
+				date,
```

Comment:
```suggestion
				span,
				date,
```

`hogehoge: hogehoge,` か `hogehoge,` どっちかに合わせたい
(というか本当はlintを入れるべきそうだけどとりあえず)

Reply:
このファイルの他の所がみんな`hogehoge: hogehoge`なのでそちらに合わした